products/opensuse/main.pm

```diff
@@ -515,6 +507,7 @@ sub load_extra_tests() {
         if (get_var("FILESYSTEM", "btrfs") eq "btrfs") {
             loadtest "console/btrfs_autocompletion";
             if (get_var("NUMDISKS", 0) > 1) {
+                loadtest "console/snapper_cleanup";
```

Comment:
https://openqa.opensuse.org/tests/340474#step/snapper_cleanup/1 shows that there is no "root" config for snapper. That probably means that the image created is so small that the installer correctly suggests to not create a filesystem with snapshots.

@asmorodskyi did you or can you crosscheck the HDD image sizes on osd/o3 regarding this?